components/Variables/VarList.tsx

```diff
@@ -0,0 +1,49 @@
+import { useContext } from "react";
+import { Var } from "./Var";
+import { VarsContext, VarsContextProps } from "./context";
```

Comment:
I prefer to import types explicitly. 
```suggestion
import { VarsContext } from "./context";
import type { VarsContextProps } from "./context";
```